package.json

```
@@ -6,7 +6,8 @@
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
     "build": "rollup --config && terser dist/orbital.js > dist/orbital.min.js && npm run measure",
-    "measure": "cat dist/orbital.min.js | gzip -9f > dist/orbital.min.gzip.js && wc -c dist/*.js"
+    "measure": "cat dist/orbital.min.js | gzip -9f > dist/orbital.min.gzip.js && wc -c dist/*.js",
+    "generate-docs": "node_modules/.bin/jsdoc --configure .jsdoc.json --verbose"
```

You don't need `node_modules/.bin` reference in NPM scripts, NPM automatically resolves these:
```suggestion
    "generate-docs": "jsdoc --configure .jsdoc.json --verbose"
```